src/seedu/addressbook/data/person/Address.java

```diff
@@ -2,18 +2,32 @@
 
 import seedu.addressbook.data.exception.IllegalValueException;
 
+
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.